core/templates/dev/head/services/ExplorationHtmlFormatterService.js

```diff
@@ -23,8 +23,21 @@ oppia.factory('ExplorationHtmlFormatterService', [
   '$filter', 'extensionTagAssemblerService', 'HtmlEscaperService',
   function($filter, extensionTagAssemblerService, HtmlEscaperService) {
     return {
+      /**
+      * @param {string} interactionId - The id of interaction in camel case.
+      * @param {array} interactionCustomizationArgSpecs - The various attributes
```

Comment:
Are you sure this is an array? How did you check?

Reply:
In extensionTagAssemblerService, I checked, and there it was used as an array.

Reply:
No, its an object, misread the for loop, will change that.